hashicorp/boundary-ui
# fix: 🐛 auth-methods deleted items are showing on table view

## Description
Bug with deleted auth-methods not being cleared from indexedDB.



## Screenshots (if appropriate):

## How to Test


1. start boundary dev instance
2. delete an auth method
3. validate auth-method is no longer displayed in table.



:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
